src/pages/AdminPages/ApprovedAccounts/index.js

```diff
@@ -0,0 +1,53 @@
+import React, {useEffect} from 'react'
+import { useSelector, useDispatch, shallowEqual } from 'react-redux'
+import Authenticate from '../../../components/Authenticate'
+import PageNotFound from '../../PageNotFound'
+import { 
+    getTraders, getUsers, getAdmins,
+    getAffiliates, setAlterUsers, setIsLoading 
+} from "../../../store/reducers/adminQuery";
+import AlterUsers from '../../../components/Admin/AlterUsers';
+
+const ApprovedAccounts = () => {
+    const dispatch = useDispatch()
+    const { role, isLoading, alterUsers, traders } = useSelector(state => ({
+        role : state.user.role,
+        isLoading : state.adminQuery.isLoading,
+        alterUsers: state.adminQuery.alterUsers,
+        traders: state.adminQuery.traders,
+    }), shallowEqual)
+
+    useEffect(() => {
+        dispatch(setIsLoading());
+        dispatch(getUsers())
+        dispatch(getTraders())
```

Comment:
If you are getting traders here in the `Approved` page (which I think is a good idea), send the traderArray as props into the `AlterUsers` component. Do same for `Unapproved` page.